### DISCUSSION TITLE
How to include optional mathjax modules

### DISCUSSION THREAD

**Discussion by boschet** [dendronhq/dendron#673] [Q&A]
I tried to include the `siunitx` into mathjax preview. But as I don't know anything about how this works in dendron/markdown preview enhanced I'd like to ask if somebody has already done that and knows how to do that. 
Specifically: How to use the mathjax_config.js?

**Comment by ghost** [comment]
You can edit the `mathjax_config.js` through the command  `Markdown Preview Enhanced: Open MathJax Config` if you mean the one for the preview inside of VSCode.

**Reply by boschet** [reply]
Yes, I tried that, but unfortunately I had no success. I also tried to extend it to load all official extensions, but no success either.

My config looks something like that:

``` js
module.exports = {
  extensions: ['tex2jax.js', '[siunitx]/siunitx.js'],
  jax: ['input/TeX-full','output/HTML-CSS'],
  messageStyle: 'none',
  tex2jax: {
    processEnvironments: false,
    processEscapes: true
  },
  TeX: {
    extensions: ['AMSmath.js', 'AMSsymbols.js', 'noErrors.js', 'noUndefined.js', 'siunitx.js']
  },
  'HTML-CSS': { availableFonts: ['TeX'] }
};
MathJax.Ajax.config.path['siunitx']  = 'http://ghcdn.rawgit.org/burnpanck/MathJax-siunitx/master/';

```
